.flake8

```diff
@@ -5,7 +5,7 @@ max-line-length = 88
 per-file-ignores =
     tests/*:S101
     tests/dwd/radar/test_index.py:E501,B950,S101
-    wetterdienst/__init__.py:F401
+    __init__.py:F401
```

Comment:
Has this been intentional? Does that mean it will apply to all `__init__.py` files?

Reply:
Exactly that's the intention, as we have several init files only used to structure the submodule

Reply:
I see, thanks. Then, we might want to remove the individual `# noqa:F401` comments within all of the `__init__.py` files?

Reply:
I think I did that already ;)